x/ibc/23-commitment/README.md

```diff
@@ -0,0 +1,50 @@
+# ICS 23: Commitment
```

Comment:
this should be moved to a godoc instead of a README and it should include a link to the ICS23 spec